src/test/java/org/broadinstitute/hellbender/tools/copynumber/CollectAllelicCountsIntegrationTest.java

```diff
@@ -70,37 +69,23 @@ public void test(final File inputBAMFile,
         final File outputFile = createTempFile("collect-allelic-counts-test-output", ".tsv");
         final String[] arguments = {
                 "-" + StandardArgumentDefinitions.INPUT_SHORT_NAME, inputBAMFile.getAbsolutePath(),
-                "-" + ExomeStandardArgumentDefinitions.SITES_FILE_SHORT_NAME, SITES_FILE.getAbsolutePath(),
+                "-L", SITES_FILE.getAbsolutePath(),
                 "-" + StandardArgumentDefinitions.REFERENCE_SHORT_NAME, REF_FILE.getAbsolutePath(),
                 "-" + StandardArgumentDefinitions.OUTPUT_SHORT_NAME, outputFile.getAbsolutePath()
         };
         runCommandLine(arguments);
         final AllelicCountCollection countsResult = new AllelicCountCollection(outputFile);
+        Assert.assertEquals(countsExpected.getCounts().size(), countsResult.getCounts().size());
         Assert.assertEquals(countsExpected, countsResult);
     }
 
     //Regression test for https://github.com/broadinstitute/gatk-protected/issues/373
-    @Test(expectedExceptions = UserException.class)
-    public void testNonStrictBAM() {
-        final File outputFile = createTempFile("collect-allelic-counts-test-output", ".tsv");
-        final String[] arguments = {
-                "-" + StandardArgumentDefinitions.INPUT_SHORT_NAME, NON_STRICT_BAM_FILE.getAbsolutePath(),
-                "-" + ExomeStandardArgumentDefinitions.SITES_FILE_SHORT_NAME, SITES_FILE.getAbsolutePath(),
-                "-" + StandardArgumentDefinitions.REFERENCE_SHORT_NAME, REF_FILE.getAbsolutePath(),
-                "-" + StandardArgumentDefinitions.OUTPUT_SHORT_NAME, outputFile.getAbsolutePath(),
-                "-" + StandardArgumentDefinitions.READ_VALIDATION_STRINGENCY_SHORT_NAME, ValidationStringency.STRICT.toString()
-        };
-        runCommandLine(arguments);
-        //should catch SAMFormatException and throw new UserException with --readValidationStringency STRICT
-    }
-
-    //Regression test for https://github.com/broadinstitute/gatk-protected/issues/373
     @Test
```

Comment:
@cmnbroad Can you just comment before I start deleting this test?

@samuelklee FYI... an index is required for this tool, so you cannot use sam files, since they cannot have indices.